src/locale/languages/es.js

```diff
@@ -49,261 +51,272 @@ export const erase = {
-      codePushUpdated: 'App is up to date',
-      codePushAvailable: 'Update is available!',
-      codePushNotApplicable: 'Not applicable in debug mode',
+      orgsRequestApprovalTop: '¿No ves todas tus organizaciones?',
+      orgsRequestApprovalBottom: 'Quizá tengas que solicitar aprobación para verlas.',
+      codePushCheck: 'Comprobar actualizaciones',
+      codePushChecking: 'Comprobando actualizaciones...',
+      codePushUpdated: 'Aplicación actualizada',
+      codePushAvailable: 'Actualizaciones disponibles!',
+      codePushNotApplicable: 'No aplicable en modo de depuración',
     },
     userOptions: {
+      donate: 'Realizar donación',
       title: 'Opciones',
-      language: 'Lenguaje',
+      language: 'Idioma',
       privacyPolicy: 'Política de privacidad',
-      signOut: 'Sign Out',
+      signOut: 'Cerrar sesión',
     },
     privacyPolicy: {
-      title: 'Política de Privacidad',
-      effectiveDate: 'Last updated: July 15, 2017',
+      title: 'Política de privacidad',
+      effectiveDate: 'Última actualización: 15 de Julio, 2017',
       introduction:
-        "We're glad you decided to use GitPoint. This Privacy Policy is here to inform you about what we do — and do not do — with our user's data.",
-      userDataTitle: 'USER DATA',
+        'Estamos contentos de que hayas decidido usar GitPoint. Esta política de privacidad está aquí para informarte sobre lo que hacemos - y lo que no - con tus datos de usuario.',
+      userDataTitle: 'DATOS DE USUARIO',
       userData1:
-        "We do not do anything with your GitHub information. After authenticating, the user's OAuth token is persisted directly on their device storage. It is not possible for us to retrieve that information. We never view a user's access token nor store it whatsoever.",
+        'No hacemos nada con tu información de GitHub. Tras la autenticación, el token OAuth del usuario se guarda directamente en el almacenamiento de su dispositivo. No es posible para nosotros recuperar esa información. Nunca vemos ni almacenamos el token de acceso de un usuario.',
       userData2:
-        "This means that in no way, shape or form do we ever view, use or share a user's GitHub data. If private data ever becomes visible at any point we will not record or view it. If it happens to be accidentally recorded, we will delete it immediately using secure erase methods. Again, we've set up authentication specifically so that this never happens.",
-      analyticsInfoTitle: 'ANALYTICS INFORMATION',
+        'Esto significa que de ninguna manera miramos, usamos o compartimos datos de GitHub de los usuarios. Si la información privada se volviera visible en algún momento, no la registraremos ni leeremos. Si se guardara accidentalmente, la borraremos inmediatamente usando métodos seguros de borrado. Sin embargo, hemos configurado la autenticación para que esto nunca suceda.',
+      analyticsInfoTitle: 'INFORMACIÓN ANALÍTICA',
```

Comment:
I think `Información de Analytics` would be better.

Reply:
@khvilaboa can you please update this?

Reply:
@alejandronanez  Updated